Dockerfile

```diff
@@ -1,14 +1,40 @@
-FROM gradle:jdk8 as builder
+FROM gradle:jdk8 as cache
+RUN mkdir -p /home/gradle/cache
+ENV GRADLE_USER_HOME /home/gradle/cache
+COPY build.gradle /home/gradle/src/
+WORKDIR /home/gradle/src
+RUN gradle clean dependencies -i --stacktrace
```

Comment:
As the cache stage is mainly used to cache the dependencies during the development, I think this stage can be removed in the final version of this Dockerfile.